apbs/use-shared-services-across-namespaces.md

```diff
@@ -0,0 +1,54 @@
+# Abstract
+
+APBs for resource intensive services should be programmed to re-use the service that they would otherwise deploy, if that service is already deployed and available in another namespace.
```

Comment:
might want to add a little more to the definition (the service need to support being shared) For instance at the moment I don't think there is anyway to share the sync service